src/sequence-steps.js

```diff
@@ -0,0 +1,231 @@
+/* eslint no-console: 0 */
+
+import utils from "./utils";
+import nodefn from "when/node";
+import semver from "semver";
+
+const CHANGELOG_PATH = "./CHANGELOG.md";
+const sequenceSteps = [
+	gitFetchUpstreamMaster,
+	gitCheckoutMaster,
+	gitMergeUpstreamMaster,
+	gitMergeUpstreamDevelop,
+	updateVersion,
+	gitLog,
+	updateLog,
+	updateChangelog,
+	gitDiff,
+	gitAdd,
+	gitCommit,
+	gitTag,
+	gitPushUpstreamMaster,
+	npmPublish,
+	gitCheckoutDevelop,
+	gitMergeMaster,
+	gitPushUpstreamDevelop,
+	gitPushOriginMaster
+];
+
+export function gitFetchUpstreamMaster( [ git, options ] ) {
+	const command = "git fetch upstream --tags";
```

Comment:
This used to be "git fetch upstream master --tags", but now it is just "git fetch upstream --tags"